examples/gcoap/Makefile

```diff
@@ -35,6 +35,23 @@ DEVELHELP ?= 1
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
+# Enables DTLS-secured CoAP messaging
+GCOAP_ENABLE_DTLS ?= 1
+ifeq (1,$(GCOAP_ENABLE_DTLS))
+	# Required by DTLS. Currently, only tinyDTLS is supported by sock_dtls.
+	USEPKG += tinydtls
+	USEMODULE += tinydtls_sock_dtls
+	USEMODULE += gcoap_dtls
+	# tinydtls needs crypto secure PRNG
+	USEMODULE += prng_sha1prng
+
+	CFLAGS += -DCONFIG_GCOAP_ENABLE_DTLS=1
+	# Maximum number of DTLS sessions
+	CFLAGS += -DDTLS_PEER_MAX=1
+	# increase main stack
+	CFLAGS += -DTHREAD_STACKSIZE_MAIN=2*THREAD_STACKSIZE_DEFAULT
```

Comment:
is it not enough to bump `GCOAP_STACK_SIZE`?

Reply:
The requests are sent from the main thread. Back then, I always had to increase the main stack because it wasn't quite enough. Tested it again, and it seems that no increase is necessary anymore. But I lack boards to be able to test it more extensively. However, if it is not enough on certain boards, this should quickly become noticeable.